packages/reports/src/Containers/open-positions.tsx

```diff
@@ -655,8 +655,7 @@ const OpenPositions = observer(({ component_icon, ...props }: TOpenPositions) =>
             />
         );
     };
-    // TODO: Uncomment and update this when DTrader 2.0 development starts:
-    // if (useFeatureFlags().is_dtrader_v2_enabled) return <Text size='l'>I am Open positions for DTrader 2.0.</Text>;
```

Comment:
it's unlikely that we're gonna do the redesign of reports under dtrader_v2 flag. this comment doesn't seem necessary.